src/logging.js

```diff
@@ -87,25 +82,45 @@ type Loggers = {
 };
 
 function initLogging(logLevel: LogLevel, isDev: boolean): Loggers {
+    // This is the logger that we use to log general information in our app.
+    // Whereever one might use console, use this instead.
+    const winstonLogger = winston.createLogger<NpmLogLevels>({
+        level: logLevel,
+        transports: getTransports(isDev),
+    });
+
     // This is the logger that captures requests handled by our express server.
     const requestLogger = expressWinston.logger({
-        level: `${logLevel}`,
-        transports: getTransports(false, isDev),
+        /**
+         * Specify the level that this logger logs at.
+         * (use a function to dynamically change level based on req and res)
+         *     `function(req, res) { return String; }`
+         */
+        level: "info",
```

Comment:
This was set to `logLevel` which was not the right thing. I have changed it add added commentary. Similarly for the error logger below.